db/migrations/20220220213655_add_unique_constraint_isbn.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+  return knex.schema.alterTable('favorites', (table) => {
+    table.unique('isbn')
```

Comment:
Good find!  I am so happy this worked in preventing multiple of the same to be added to a table